### PR TITLE
Remove 'Full nomination will be available in...' (bug 1200790)

### PR DESCRIPTION
**Pull request by wagnerand**
@magopian r?

### CODE REVIEW

**Comment by wagnerand** [comment]
Updated to drop the feature entirely.

File: apps/addons/models.py [hunk 1]
```diff
@@ -1183,32 +1182,10 @@ def can_request_review(self):
         elif self.status == amo.STATUS_UNREVIEWED:
             return (amo.STATUS_PUBLIC,)
         elif self.status == amo.STATUS_LITE:
-            if self.days_until_full_nomination() == 0:
-                return (amo.STATUS_PUBLIC,)
-            else:
-                # Still in preliminary waiting period...
-                return ()
+            return (amo.STATUS_PUBLIC,)
```
Comment (magopian) [inline apps/addons/models.py]: Merge this with the previous `elif`?

**Comment by magopian** [comment]
Excellent work, thanks! r+wc

**Comment by wagnerand** [comment]
Done, thank you!

**Pull request merged**